pkg/machine/qemu/machine.go

```diff
@@ -647,6 +648,35 @@ func getDevNullFiles() (*os.File, *os.File, error) {
 	return dnr, dnw, nil
 }
 
+// qemuPid returns -1 or the PID of the running QEMU instance.
+func (v *MachineVM) qemuPid() (int, error) {
+	pidData, err := os.ReadFile(v.VMPidFilePath.GetPath())
+	if err != nil {
+		if !errors.Is(err, os.ErrNotExist) {
```

Comment:
Is this correct, we do not want to return errors other the os.ErrNotExists?

Reply:
Ah, now I got it.  Moving the code around messed up the logic.